frontend/openchat-agent/src/services/openchatAgent.ts

```diff
@@ -641,6 +642,48 @@ export class OpenChatAgent extends EventTarget {
         }
     }
 
+    // chatEventsWindow(
```

Comment:
Are these meant to be commented out?

Reply:
Ahh these are newly added... I thought you'd commented out a load of existing code.

These can all be deleted.